src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -172,18 +173,27 @@ Amplify.configure({
 ```
 
 ### Recap
-At this point your client is now configured to point to your AppSync API endpoint and you updated your authentication details based on the auth types you are using.
+At this point, your client is now configured to point to your AppSync API endpoint and you updated your authentication details based on the auth types you are using.
 
 [comment]: # (ADD TO RECAP Tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
 
 ## Connect your application code to the data backend
 
```

Comment:
This section reads out of place as written. It needs a better tie in to the intro above and what follows. From the intro I was expecting to dive right into connecting and configuring libraries not using codegen. Can we update the intro and this section to work together more so it reads less out of place?